waku/v2/protocol/waku_swap/waku_swap_types.nim

```diff
@@ -6,6 +6,14 @@ import
   ../../node/peer_manager/peer_manager  
 
 type
+   #The Swap Mode determines the functionality available in the swap protocol.
```

Comment:
Make sure your indentation is consistent here. 
Also, very pedantic from my side, but if there is a space between the `#` and the first word in that comment, it becomes easier to read, spell-check and add to docs. 😃

Reply:
Thanks for the review. The spacing format is worth adhering to. ☀︎